src/main/scala/com/microsoft/hyperspace/index/rules/FilterIndexRule.scala

```diff
@@ -120,10 +125,21 @@ object FilterIndexRule extends Rule[LogicalPlan] with Logging {
         val newOutput =
           logicalRelation.output.filter(attr => index.schema.fieldNames.contains(attr.name))
 
-        project.copy(
+        val updatedPlan = project.copy(
           child =
             filter.copy(child = logicalRelation.copy(relation = newRelation, output = newOutput)))
 
+        // TODO: implement scrubber to remove PII from plans before adding plans to events.
```

Comment:
why? it should be the logger implementation which should worry about this.

Reply:
Oh yes that's correct. Let me fix this one. Thanks